### PR TITLE
doctl-databases-*: add page

### PR DESCRIPTION
**Pull request by kicka5h**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**
doctl v1.98.1

### CODE REVIEW

File: pages/common/doctl-databases-db.md
Comment (reinhart1010) [inline pages/common/doctl-databases-db.md:6]:
```suggestion
- Run a `doctl databases db` command with an access token:
```
Use backticks to clarify that these sre executable commands.

File: pages/common/doctl-databases-db.md
Comment (reinhart1010) [inline pages/common/doctl-databases-db.md:8]:
```suggestion
`doctl databases db [command] --access-token {{access_token}}`
```

No leading and trailing spaces on placeholder names, [like our style guide](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#placeholder-syntax).

File: pages/common/doctl-databases-db.md
Comment (reinhart1010) [inline pages/common/doctl-databases-db.md:12]:
```suggestion
`doctl databases db get {{database_id}} {{db_name}}`
```
[Our style guide](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#placeholder-syntax) mandates that placeholders_are_written_in_snake_case.

File: pages/common/doctl-databases-db.md
Comment (reinhart1010) [inline pages/common/doctl-databases-db.md:16]:
```suggestion
`doctl databases db list {{database_id}}`
```

File: pages/common/doctl-databases-db.md
Comment (reinhart1010) [inline pages/common/doctl-databases-db.md:20]:
```suggestion
`doctl databases db create {{database_id}} {{db_name}}`
```

File: pages/common/doctl-databases-db.md
Comment (reinhart1010) [inline pages/common/doctl-databases-db.md:24]:
```suggestion
`doctl databases db delete {{database_id}} {{db_name}}`
```

**Review by reinhart1010 — CHANGES_REQUESTED** [review]
Thanks for contributing! Before I and other maintainers can approve, please fix the following writing style issues (and check out our [style guide](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#placeholder-syntax) for reference):

+ Use backticks (\`...\`, aka. Markdown inline code syntax) for highlighting code in example descriptions, such as "run a `doctl databases` command…" instead of "run a doctl databases command…".
+ When writing placeholders like `{{this}}`, **do not** add leading and trailing space to the placeholder names, like `{{ this }}`.
+ `{{long_variables_like_this_should_all_be_written_in_snake_case}}`.

I've written some examples for the `doctl databases db` subcommand; please revise the other pages too.

File: pages/common/doctl-databases-maintenance-window.md
Comment (reinhart1010) [inline pages/common/doctl-databases-maintenance-window.md:8]:
```suggestion
`doctl databases maintenance-window {{command}} --access-token {{access_token}}`
```

File: pages/common/doctl-databases-options.md
Comment (reinhart1010) [inline pages/common/doctl-databases-options.md:8]:
```suggestion
`doctl databases options {{command}} --access-token {{access_token}}`
```

File: pages/common/doctl-databases-pool.md
Comment (reinhart1010) [inline pages/common/doctl-databases-pool.md:8]:
```suggestion
`doctl databases pool {{command}} --access-token {{access_token}}`
```

File: pages/common/doctl-databases-replica.md
Comment (reinhart1010) [inline pages/common/doctl-databases-replica.md:8]:
```suggestion
`doctl databases pool {{command}} --access-token {{access_token}}`
```

File: pages/common/doctl-databases-sql-mode.md
Comment (reinhart1010) [inline pages/common/doctl-databases-sql-mode.md:8]:
```suggestion
`doctl databases sql-mode {{command}} --access-token {{access_token}}`
```

File: pages/common/doctl-databases-user.md
Comment (reinhart1010) [inline pages/common/doctl-databases-user.md:8]:
```suggestion
`doctl databases user {{command}} --access-token {{access_token}}`
```

File: pages/common/doctl-databases.md
Comment (reinhart1010) [inline pages/common/doctl-databases.md:8]:
```suggestion
`doctl databases {{command}} --access-token {{access_token}}`
```

**Review by reinhart1010 — CHANGES_REQUESTED** [review]
Just another minor change and you're done.

**Review by reinhart1010 — APPROVED** [review]
Thanks! We'll need another contributor to approve and merge this PR.

File: pages/common/doctl-databases-db.md
Comment (gutjuri) [inline pages/common/doctl-databases-db.md:4]:
```suggestion
> More information: <https://docs.digitalocean.com/reference/doctl/reference/databases/db>.
```

File: pages/common/doctl-databases-firewalls.md
Comment (gutjuri) [inline pages/common/doctl-databases-firewalls.md:4]:
```suggestion
> More information: <https://docs.digitalocean.com/reference/doctl/reference/databases/firewalls>.
```

File: pages/common/doctl-databases-firewalls.md
Comment (gutjuri) [inline pages/common/doctl-databases-firewalls.md:16]:
```suggestion
`doctl databases firewalls append {{database_id}} --rule {{droplet|k8s|ip_addr|tag|app}}:{{value}}`
```

see https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#grouping-placeholders

File: pages/common/doctl-databases-maintenance-window.md
Comment (gutjuri) [inline pages/common/doctl-databases-maintenance-window.md:4]:
```suggestion
> More information: <https://docs.digitalocean.com/reference/doctl/reference/databases/maintenance-window>.
```

File: pages/common/doctl-databases-maintenance-window.md
Comment (gutjuri) [inline pages/common/doctl-databases-maintenance-window.md:16]: not sure if we can have `e.g.` in placeholders. @kbdharun what is your opinion on this?

File: pages/common/doctl-databases-options.md
Comment (gutjuri) [inline pages/common/doctl-databases-options.md:4]:
```suggestion
> More information: <https://docs.digitalocean.com/reference/doctl/reference/databases/options>.
```

File: pages/common/doctl-databases-options.md
Comment (gutjuri) [inline pages/common/doctl-databases-options.md:16]:
```suggestion
`doctl databases options regions --engine {{pg|mysql|redis|mongodb}}`
```

File: pages/common/doctl-databases-options.md
Comment (gutjuri) [inline pages/common/doctl-databases-options.md:20]:
```suggestion
`doctl databases options slugs --engine {{pg|mysql|redis|mongodb}}`
```

File: pages/common/doctl-databases-options.md
Comment (gutjuri) [inline pages/common/doctl-databases-options.md:24]:
```suggestion
`doctl databases options versions --engine {{pg|mysql|redis|mongodb}}`
```

File: pages/common/doctl-databases-pool.md
Comment (gutjuri) [inline pages/common/doctl-databases-pool.md:4]:
```suggestion
> More information: <https://docs.digitalocean.com/reference/doctl/reference/databases/pool>.
```

File: pages/common/doctl-databases-replica.md
Comment (gutjuri) [inline pages/common/doctl-databases-replica.md:4]:
```suggestion
> More information: <https://docs.digitalocean.com/reference/doctl/reference/databases/replica>.
```

File: pages/common/doctl-databases-sql-mode.md
Comment (gutjuri) [inline pages/common/doctl-databases-sql-mode.md:4]:
```suggestion
> More information: <https://docs.digitalocean.com/reference/doctl/reference/databases/sql-mode>.
```

File: pages/common/doctl-databases-sql-mode.md
Comment (gutjuri) [inline pages/common/doctl-databases-sql-mode.md:16]:
```suggestion
`doctl databases sql-mode set {{database_id}} {{sql_mode_1 sql_mode_2 ...}}`
```

File: pages/common/doctl-databases-user.md
Comment (gutjuri) [inline pages/common/doctl-databases-user.md:4]:
```suggestion
> More information: <https://docs.digitalocean.com/reference/doctl/reference/databases/user>.
```

File: pages/common/doctl-databases-user.md
Comment (gutjuri) [inline pages/common/doctl-databases-user.md:20]:
```suggestion
`doctl databases user reset {{database id}} {{user_name}}`
```

File: pages/common/doctl-databases-user.md
Comment (gutjuri) [inline pages/common/doctl-databases-user.md:24]:
```suggestion
`doctl databases user reset {{database_id}} {{user_name}} {{caching_sha2_password|mysql_native_password}}`
```

File: pages/common/doctl-databases.md
Comment (gutjuri) [inline pages/common/doctl-databases.md:4]:
```suggestion
> More information: <https://docs.digitalocean.com/reference/doctl/reference/databases>.
```

**Review by gutjuri — CHANGES_REQUESTED** [review]
Some more formatting errors and typos.

Thank you very much for your contribution!

File: pages/common/doctl-databases-pool.md
Comment (kbdharun) [reply pages/common/doctl-databases-pool.md:4]: I am resolving this suggestion as the site resolves to the address with `/`. We only remove it in cases when the site doesn't redirect to a URL with `/`.

**Review by gutjuri — APPROVED** [review]

File: pages/common/doctl-databases-maintenance-window.md
Comment (kbdharun) [reply pages/common/doctl-databases-maintenance-window.md:16]:
```suggestion
`doctl databases maintenance-window update {{database_id}} --day {{day_of_the_week}} --hour {{hour_in_24_hours_format}}`
```

File: pages/common/doctl-databases-db.md
Comment (kbdharun) [inline pages/common/doctl-databases-db.md:24]:
```suggestion
`doctl databases db get {{database_id}} {{database_name}}`

- List existing databases hosted within a given database cluster:

`doctl databases db list {{database_id}}`

- Create a database with the given name in the given database cluster:

`doctl databases db create {{database_id}} {{database_name}}`

- Delete the database with the given name in the given database cluster:

`doctl databases db delete {{database_id}} {{database_name}}`
```

**Review by kbdharun — APPROVED** [review]
LGTM, Welcome to tldr and thanks for your contribution with this massive PR ❤️ .

**Pull request merged**
